From: =?utf-8?b?TWFya28gTcOka2Vsw6Q=?= <marko.makela@mariadb.com>
Date: Wed, 12 Nov 2025 09:18:55 +0200
Subject: MDEV-35810 test log_file_size_online fixup

A KILL QUERY of SET GLOBAL innodb_log_file_size would sometimes lead to
ER_QUERY_INTERRUPTED being reported.

This also fixes MDEV-38287 that was visible on Launchpad.net ppc64el and
s390x builders.

Origin: https://github.com/MariaDB/server/commit/8794f8c3d7e1a7dd8e1e2f8c93298ff3f8dbef95
---
 mysql-test/suite/innodb/t/log_file_size_online.test | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mysql-test/suite/innodb/t/log_file_size_online.test b/mysql-test/suite/innodb/t/log_file_size_online.test
index fb6722e..8ea5e66 100644
--- a/mysql-test/suite/innodb/t/log_file_size_online.test
+++ b/mysql-test/suite/innodb/t/log_file_size_online.test
@@ -43,6 +43,7 @@ send SET GLOBAL innodb_log_file_size=7340032;
 let $ignore= `SELECT @id := $ID`;
 KILL QUERY @id;
 --connection con1
+--error 0,ER_QUERY_INTERRUPTED
 reap;
 
 --connection default
